frontend/src/app/layouts/user-activation/reg-confirm/reg-confirm.component.ts

```diff
@@ -0,0 +1,46 @@
+import {Component, OnInit} from '@angular/core';
+import {ActivatedRoute, Router} from '@angular/router';
+import {UserActivationService} from '../user-activation.service';
+
+@Component({
+    selector: 'app-reg-confirm',
+    templateUrl: './reg-confirm.component.html',
+    styleUrls: ['./reg-confirm.component.scss']
+})
+export class RegConfirmComponent implements OnInit {
+    token: string;
+
+    constructor(private route: ActivatedRoute,
+                private router: Router,
+                private userActivationService: UserActivationService) {
+    }
+
+    ngOnInit() {
+        this.route.queryParamMap
+            .subscribe(
+                (queryParams) => {
+                    this.token = queryParams.get('token');
+                    this.activateUser(this.token);
+                });
+    }
+
+    activateUser(token: string) {
+        console.log(this.token);
```

Comment:
I think console.log does not have to be here

Reply:
corrected